README.md

```diff
@@ -99,6 +99,19 @@ use { "johmsalas/text-case.nvim",
     "ga", -- Default invocation prefix
     { "ga.", "<cmd>TextCaseOpenTelescope<CR>", mode = { "n", "v" }, desc = "Telescope" },
   },
+  -- cmd is only needed if lazy = true so that the commands can be executed even if the plugin
+  -- is not loaded yet.
+  cmd = {
+    "Subs",
```

Comment:
Perhaps a command reminding `Subs` is customizable?

Reply:
Done